research/src/pages/tabs.proposal.mdx

```diff
@@ -14,7 +14,22 @@ showInMenu: false
+
+As toolkits and operating systems evolved they also developed several specializations  with various additional qualities and different guidance on when they should be used.  Tabbed document interfaces, for example, were created as a way to mange windows in an applicaion and therefore needed the qualities of a window (the ability to move it, close it, derive its title and associated icon, and so on), simply drawn in an interface that "looked" like tab groups.  Ribbons (as some would think of them today) combined ideas of tab and (sometimes scrollable) toolbar interfaces which can sometimes be minimized (hidden), but not dismissed.  Mobile interfaces also introduced intersting new challenges.  There are several components with very similar base needs which are somewhat fluid and constantly evolving in systems in terms of exactly what they are called or what visual criteria distinquish them (accordions and tabs, for example, are differentiated in some toolkits and systems in various ways - sometimes functional in which the important bit is in how many can be active at once, and sometimes purely visual).
+
+Because this was all true while the WHATWG took up HTML 5, and there were even markup oriented expressions of several of these components in systems like XUL, XAML and Flex,  the HTML specification briefly included an element known as [`<switch>`](https://whatwg.org/specs/web-apps/2006-01-01/#switch) but this was removed in 2006.  
+
+In 2015, several W3C participants got together to try to [propose work toward a standard solution through WICG](https://discourse.wicg.io/t/panels-and-panelsets/1184) for several of these use cases.  While a fair amount of the accessibility community was interested in pursuing this, the effort ultimately failed for a number of reasons, technical limitations at the time (no wide Shadow DOM availability and the rise of frameworks with their own lifecycles greatly increased the difficulties of a really great implementation).  While it got use and feedback, it was quite limited and after some experience, eventually dropped as a proposal.  
```

Comment:
I think I'd prefer for this to reference, especially given the prior paragraph to reference that many component libraries have built tab components and... next sentence regarding developer desire